tests/buttons/tests/01-run.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect_exact("On-board button test")
+    index = child.expect([
+        r"\[FAILED\] no buttons available!",
+        r" -- Available buttons: \d+"
+    ])
+
+    if index == 1:
```

Comment:
What if the board has more buttons?

Reply:
Sorry ... this isn't the result of the regex, but the result of the branching. So all is correct :-)

Reply:
What has `index` to do with the number of buttons?

Reply:
Nothing. I was just confused ;-)